spec/rubocop/cop/rspec/unexpected_requires_spec.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::RSpec::UnexpectedRequires do
+  context 'when there is an require' do
+    it 'registers an offense' do
+      expect_offense(<<~RUBY)
+        require "base64"
+        ^^^^^^^ Do not require anything in a test file.
+
+        describe "MyTest" do
+          do_something
+        end
+      RUBY
+    end
+
+    context 'with the file being spec_helper' do
+      it 'does not register an offense' do
+        expect_no_offenses(<<~RUBY, "spec_helper.rb")
```

Comment:
The file name should match one of the include patterns in https://github.com/rubocop/rubocop-rspec/blob/e02576f1a8c39d9dc5b3ca37fea4596f7e75f493/config/default.yml#L6-L8, or the example won’t reach your cop code at all.
```suggestion
        expect_no_offenses(<<~RUBY, "spec/spec_helper.rb")
```